colp_build/sql/create_colp.sql

```diff
@@ -191,7 +201,7 @@ pluto_merge AS (
         END) as cd
     FROM sname_merge a 
     LEFT JOIN dcp_pluto b
-    ON a.bbl::text = b.bbl::text
```

Comment:
I know in emails it's worded backfilling missing cd with pluto CD, but I wonder if the intent is to just take the CD from PLUTO and not from IPIS even if it is available.  I'll discuss this with LS at the end of day.